phoonnx/model_manager.py

```
@@ -0,0 +1,306 @@
+                voice_info = TTSModelInfo(
+                    voice_id="mimic3_" + k,
+                    lang=lang,
+                    config_url=config_url,
+                    tokens_url=tokens_url,
+                    model_url=model_url,
+                    phoneme_map_url=phoneme_map_url
+                )
+                voice_info.config.lang = lang
+                voice_info.config.speaker_id_map = speaker_map
+                self.add_voice(voice_info)
+            except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)